cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -44,8 +44,9 @@ done
 
 set -e
 
-if [ "$NETWORK_PROVIDER" == "weave" ]; then 
-  kubectl apply -f https://github.com/weaveworks/weave/releases/download/v1.9.4/weave-daemonset-k8s-1.6.yaml
+if [ "$NETWORK_PROVIDER" == "weave" ]; then
+  kubever=$(kubectl version | base64 | tr -d '\n')
+  kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
```

Comment:
That is nice!

Reply:
@rmohr From here https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network :smile: